Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -119,6 +119,33 @@ of removing unused segments after release of internal resources.
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
 
```

Comment:
"ArangoSearch performs..." (below) is meant as description for the following properties, and not related to the `consolidationIntervalMsec` option, right? I think it would be better in this case to have this as normal text and start another list below, which is actually a continuation of above list.